### ISSUE TITLE
Minor issue in XMP format in sidecar - dc:subject

### ISSUE THREAD

**Issue by narensankar0529** [RhetTbull/osxphotos#349]
<dc:subject> - The RDF generated by osxphotos is tagged as Seq instead of Bag.

This means that the keyword order is important v/s unordered as it is typically used all almost all applications.

Suggest change Seq to Bag

More information - https://www.dublincore.org/specifications/dublin-core/dcq-rdf-xml/

For example change - 

<dc:subject>
            <rdf:Seq>
                <rdf:li>Founding of the Republic of China</rdf:li>
                <rdf:li>Republic Day &amp; New Year’s Day</rdf:li>
                <rdf:li>Winter</rdf:li>
                <rdf:li>bowl</rdf:li>
                <rdf:li>plate</rdf:li>
                <rdf:li>spoon</rdf:li>
                <rdf:li>tableware</rdf:li>
                <rdf:li>utensil</rdf:li>
                <rdf:li>XXXXXXXX</rdf:li>
                <rdf:li>YYYYYY</rdf:li>
            </rdf:Seq>
        </dc:subject>

TO -

 <dc:subject>
   <rdf:Bag>
    <rdf:li>Founding of the Republic of China</rdf:li>
    <rdf:li>Republic Day &amp; New Year’s Day</rdf:li>
    <rdf:li>Winter</rdf:li>
    <rdf:li>bowl</rdf:li>
    <rdf:li>plate</rdf:li>
    <rdf:li>spoon</rdf:li>
    <rdf:li>tableware</rdf:li>
    <rdf:li>utensil</rdf:li>
    <rdf:li>XXXXXXXX</rdf:li>
    <rdf:li>YYYYYY</rdf:li>
   </rdf:Bag>
  </dc:subject>

**Comment by RhetTbull** [comment]
Thanks.  I'll take a look this weekend.  Interestingly, to make the XMP template for osxphotos, I copied (literally) what Photos does.  I just checked again and, at least on Catalina, Photos stills outputs subject as a Seq:
```XML
         <dc:subject>
            <rdf:Seq>
               <rdf:li>UK</rdf:li>
               <rdf:li>London 2018</rdf:li>
               <rdf:li>St. James's Park</rdf:li>
               <rdf:li>United Kingdom</rdf:li>
               <rdf:li>London</rdf:li>
               <rdf:li>England</rdf:li>
            </rdf:Seq>
         </dc:subject>
```

**Comment by narensankar0529** [comment]
Seq is not wrong. I guess it depends on the application. Exiftool prefers Bag and so does Lightroom and various other apps I looked at. Mainly because the assumption is that the Keywords are not ordered or prioritized. So either way is ok. If you want to match Photos output that is ok as well. This is not an error overall in any way. I did not check the output from Photos, but from exiftool and Lightroom hence reported the difference. Perhaps just documenting it might also be ok.

**Comment by RhetTbull** [comment]
I'd prefer the output is as compatible as possible with other apps so if exiftool and Lightroom prefer Bag, that's an easy fix.  I don't personally use XMP files and have no way to easily test these so I appreciate the feedback from someone who is using them.

**Comment by narensankar0529** [comment]
Exiftool corrects it automatically and prints a warning - Warning: [minor] Fixed incorrect list type for XMP-dc:Subject

So if you can change it to Bag it would be great!

**Comment by RhetTbull** [comment]
I think this is fixed in [v0.39.18](https://github.com/RhetTbull/osxphotos/releases/tag/v0.39.18).  I've fixed XMP sidecar output to match exiftool as much as possible.  Let me know if you still have warnings.

**Comment by narensankar0529** [comment]
Works great! Thanks a lot
